.github/workflows/publish.yml

```diff
@@ -0,0 +1,51 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      package:
+        description: The package to release, this should be the path segment after packages/, without the @sapphire/ prefix
+        required: true
+
+jobs:
+  PublishPackage:
+    name: Publish @sapphire/${{ github.event.inputs.package}}
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'sapphiredev'
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: main
+          token: ${{ secrets.SKYRA_TOKEN }}
```

Comment:
Why don't you clone with the github_token, if you use it for literally everything else down below?

Reply:
Because then you run into not being allowed to push to main: https://github.com/favware/cliff-jumper/actions/runs/9353623553/workflow

Reply:
But you literally change the origin URL to the other token?

Reply:
Doesn't take away that without the CI run fails as shown above. How the origin URL plays into that I don't know. I also tried removing setting the URL but that too failed. This is the way I've been running automatic updates for graphql-pokemon for a very long time now.